docs/pages/api-reference/aggregations.md

```diff
@@ -15,7 +15,11 @@ Besides these common arguments, here is the rest of the API reference for all th
 
 ### 1. Count
 
-Count computes a rolling count for each group key across a window. Takes no additional arguments besides `window` and `into_field`. It returns 0 by default. Its output type is always `int`.&#x20;
+Count computes a rolling count for each group key across a window.  It returns 0 by default. Its output type is always `int`.&#x20;
+The count aggregate also takes an optional argument `unique` which is a boolean. If set to true, it counts the number of unique values in the given window.&#x20;
+The field over which the count is computed is specified by the `of` parameter of type `str`.&#x20;
+Since the unique count uses a HyperLogLog algorithm to estimate the number of unique values and hence is an approximate count.&#x20;
+Hence please set the `approx` parameter to true while using this feature.&#x20;
```

Comment:
This should be rewritten to have a different narrative. You can say something to the effect of "approx is another argument which when turned on makes the answer approximate but allows Fennel to be more efficient with state storage. As of right now, Fennel only supports approx unique count so if you set unique=True, you'd also need to set approx=True.